docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,308 @@
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+<Admonition type="tip">
+You can make the policy more strict by providing specific IAM role resource
+ARNs in the "Resource" field instead of using a wildcard.
+</Admonition>
+
+## Step 5/5. Connect
+
+Once the Application Service has started and joined the cluster, you can start
+connecting to your DynamoDB database.
+
+### Using AWS Management Console
+
+First log in to the Teleport Web UI at `https://teleport.example.com` (replace
+with your Proxy Service's public address).
+
+Navigate to the Applications tab in your Teleport cluster's control panel and
+click on the Launch button for the AWS DynamoDB application. This will bring up
+an IAM role selector:
+
+![IAM role selector](../../../img/database-access/guides/dynamodb-select-iam-role.png)
+
+Click on the role you want to assume and you will get redirected to the AWS
+Management Console, signed in with the selected role.
+
+In the console's top-right corner you should see that you're logged in through
+federated login and the name of your assumed IAM role:
+
+![Federated login](../../../img/database-access/guides/dynamodb-federated-login.png)
+
+Note that your federated login session is marked with your Teleport username.
+
+### Using AWS CLI
+Now, log into the previously configured AWS DynamoDB app on your desktop:
+
+```code
+$ tsh app login --aws-role ExampleTeleportDynamoDBRole aws-dynamodb
+Logged into AWS app aws. Example AWS CLI command:
+
+$ tsh aws s3 ls
+```
+
+The `--aws-role` flag allows you to specify the AWS IAM role to assume when
+accessing the AWS API. You can either provide a role name like `--aws-role
+ExampleTeleportDynamoDBRole` or a full role ARN like
+`arn:aws:iam::123456789000:role/ExampleTeleportDynamoDBRole`.
+
+Now you can use the `tsh aws` command like the native `aws` command-line tool:
+
+```code
+$ tsh aws dynamodb list-tables
+```
+
+To log out of the `aws-dynamodb` application and remove credentials:
+
+```code
+$ tsh app logout aws-dynamodb
+```
+
+### Using other DynamoDB applications
+
+First, log into the previously configured AWS DynamoDB app if you haven't
+already done so:
+
+```code
+$ tsh app login --aws-role ExampleTeleportDynamoDBRole aws-dynamodb
+```
+
+To connect your DynamoDB application, you can start either a local HTTPS proxy
+or a local AWS Service Endpoint proxy.
+
+<Tabs>
+  <TabItem label="HTTPS proxy">
+  By default, starting the AWS app proxy creates a local HTTPS proxy server
+  that forwards AWS requests to the Teleport Proxy Service, enabling you to
+  access AWS applications.
+
+  Now, use the following command to start the proxy your applications will be
+  connecting to:
+
+  ```code
+  $ tsh proxy aws -p 23456
+  Started AWS proxy on http://127.0.0.1:23456.
+
+  Use the following credentials and HTTPS proxy setting to connect to the proxy:
+    AWS_ACCESS_KEY_ID=<access-key-id>
+    AWS_SECRET_ACCESS_KEY=<secret-access-key>
+    AWS_CA_BUNDLE=<ca-bundle-path>
```

Comment:
Suggested from other PR to use a placeholder instead of some random hash. Is this way like `<xxx-xx>` ok for the placeholder? or i can do the fake version like `aaaaa-bbbb-cccc"